Makefile

```diff
@@ -126,5 +126,10 @@ dkms_install: dkms_modules_install depmod_conf_install dracut_conf_install udev_
 
 dkms_uninstall: tools_uninstall xorg_conf_uninstall udev_rules_uninstall dracut_conf_uninstall depmod_conf_uninstall dkms_modules_uninstall
 
+enable_debug:
+	./digimend-debug 1
+
+dkms_reinstall: dkms_uninstall dkms_install enable_debug
```

Comment:
Hmm, I assume this would be useful for a development loop, to quickly test your changes. However, enabling debug shouldn't be a part of a general-purpose target. After removing it, its use for debugging becomes superficial, and you can just instead have a compound command in your history to do the same:

    make dkms_uninstall dkms_install && digimend-debug 1

All-in-all, I think it's more trouble than it's worth, so could you please drop this patch?